code/datums/mutantraces.dm

```diff
@@ -264,6 +264,8 @@ ABSTRACT_TYPE(/datum/mutantrace)
 			if (!needs_oxy)
 				APPLY_ATOM_PROPERTY(M, PROP_MOB_BREATHLESS, src.type)
 			src.AH = M.bioHolder?.mobAppearance // i mean its called appearance holder for a reason
+			if (!src.dna_mutagen_banned)
```

Comment:
Won't this always overwrite with the new mutantrace or am I reading it wrong?

Reply:
This ensures that `original_mutant_race` will never be a mutantrace that is mutagen banned, so that if a lizard/cow/roach/pug were to be turned into a thrall, or any other mutagen banned mutantrace, `original_mutant_race` will store a reference to their mutantrace prior to becoming a thrall, so that they may be converted back into a lizard/cow/roach/pug.